packages/backend/src/util/handlers/createExpressSynthqlHandler.test.ts

```diff
@@ -0,0 +1,58 @@
+import { beforeAll, describe, expect, test } from 'vitest';
+import { createExpressSynthqlHandler } from './createExpressSynthqlHandler';
+import { QueryEngine } from '../..';
+import { getMockReq, getMockRes } from 'vitest-mock-express';
+
+import { DB, from } from '../../tests/pagilaDb';
+
+describe('createExpressSynthqlHandler', () => {
+    let queryEngine: QueryEngine<DB>;
+
+    beforeAll(async () => {
+        queryEngine = new QueryEngine<DB>({
```

Comment:
There's a global QueryEngine instance that you can reuse

Reply:
Understood.  Can you point me to it?

Reply:
https://github.com/synthql/synthql/blob/874b522cbd60d3b15061c4ef6efa90e80d0acfc2/packages/backend/src/tests/queryEngine.ts

Reply:
General note: this is a sign that you're not peeking into the repo. Working on a new codebase can sometimes be daunting, but the only way to get to a point where you feel familiar with the codebase is by looking into it.

A strategy that I sometimes follow is to periodically look into the codebase from a high level and slowly descend into lower levels the more you get familiar with it. Here's some concrete ideas:

High level:
1. Start by looking at the file/folder structure. Notice patterns and naming.
1. Understand the package structure. Which packages depend on which.
1. Read the docs. Notice common re-ocurring terms.
1. Read/skim the [API docs](https://synthql.github.io/SynthQL/reference/)

At this stage you're just learning the current structure, patterns and learning terminology/naming used in the project.

You're also learning where certain things are in the codebase.

Mid level:
1. Read the test/coverage reports: What things are being tested, which files have high coverage, low coverage, etc.
1. Skim the test files. 
2. Skim the type definitions.

At this stage you should already be familiar with the naming, conventions, patterns and you should have identified the most important classes/functions (e.g. QueryClient, useSynthQL, etc.) You should also be able to explain to your grandma what they do and how they relate to each other.

Low level:
1. Understand the flow of information: which functions call which functions.
2. Understand the main algorithms: e.g. how is planning implemented.

At this stage you should be able to tell how information flows throughout the project (e.g. what are the most important stages, which classes/functions are public API vs internal, how/when do they communicate with each other)


Some tools/techniques to jumpstart this process:
1. Write tests. This will force you to understand the inputs/outputs.
4. Make small refactors. This will force you to read the codebase, and get a sense of completion. Keep the refactors small as they might break things that you don't understand yet.
5. Expand the documentation: If there are bits that you don't understand, write about them and then document your writings in the docs so that others will be able to benefit from this knowledge.